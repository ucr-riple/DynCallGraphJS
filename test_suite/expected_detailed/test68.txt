{
    'global (test68.js:12:11:12:21)': [
        'getKeys (test68.js:2:1:4:2)'
    ],
    'global (test68.js:14:1:14:11)': [
        'anon (test68.js:7:9:9:6)'
    ]
}