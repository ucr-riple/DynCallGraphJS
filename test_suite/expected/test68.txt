{ '(test68.js:12:11:12:21)':
   [ '(test68.js:2:1:4:2)' ],
  '(test68.js:14:1:14:11)':
   [ '(test68.js:7:9:9:6)' ] }